phac-nml/irida-next
# Namespace group link controllers

## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR adds in new controllers for both groups and projects which handle the linking/unlinking/updating a namespace with groups. Service names and policy methods were also update to reflect these new controller names. Testing for linking/unlinking/updating a namespace with groups were moved from the groups and projects controllers into new controller tests under the groups and projects modules.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. To verify please ensure that all tests have passed and that there is good test coverage.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
